### ISSUE TITLE
sops 3.6 fails to properly decrypt .env files created using sops 3.5

### ISSUE THREAD

**Issue by akanix42** [mozilla/sops#705]
sops 3.6.0 is adding additional quotes when decrypting `.env` files from version 3.5.0, which results in a breaking change as the output is no longer correct.

This appears to be caused by this change: #622, which from what I gather was supposed to be backwards compatible with v3.5.0 files.
This issue is separate from #702 as that deals with the reverse (3.6.0 files being read by 3.5.0).

Example showing the issue:
```
❯ cat test.dev-plain.env
FOO="foo"
❯ sops -e --output test.dev-3_5.env test.dev-plain.env
❯ sops -d test.dev-3_5.env
FOO="foo"
❯ ~/Downloads/sops-v3.6.0.darwin -d test.dev-3_5.env
FOO='"foo"'
```

**Comment by autrilla** [comment]
Thanks for pointing this out. I opened #706 to handle this.

**Comment by nekonata** [comment]
v3.6.1 Why hasn't it been launched to fix this problem????????????

**Comment by autrilla** [comment]
Because sops is maintained by volunteers that do not get paid for it.

**Comment by DrPhil** [comment]
Duplicate of https://github.com/mozilla/sops/issues/702?

**Comment by ajvb** [comment]
Fixed in 3.6.1 - https://github.com/mozilla/sops/releases/tag/v3.6.1

**Comment by chrisrink10** [comment]
Reverting this change broke `sops` for any files created or converted to the 3.5.x format.
